src/viking.js

```diff
@@ -1,11 +1,89 @@
 // Soldier
-class Soldier {}
-
+class Soldier {
+    constructor(health,strength){
+this.health=health;
+this.strength=strength;
+    }
+    attack(){
+        return this.strength
+    }
+    receiveDamage(damage){
+        this.health-=damage
+    }  
+}
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name,health,strength){
+    super(name,health,strength)
```

Comment:
Viking is extending the class Soldier, this means that Viking inherit health and strength from Soldier.
So if health and strength is being inherit the only thing new that you need to declare here is the name:

```
class Viking extends Soldier {
  constructor(name, health, strength){
    super(health, strength)
    this.name = name
}

(...)
```

In the constructor you need to pass the 3 arguments, in the super you only need to pass health and strength, and then you just need to declare the variable name, since health and strength is already declared at the class Soldier.